core/Symbols.cc

```diff
@@ -2288,6 +2288,30 @@ uint32_t Method::methodShapeHash(const GlobalState &gs) const {
     return result;
 }
 
+uint32_t Field::fieldShapeHash(const GlobalState &gs) const {
+    uint32_t result = _hash(name.shortName(gs));
+
+    // TODO(jez) type aliases and class aliases participate in the constant resolution fixed
+    // point, which could probably make something start to resolve in a file that doesn't
+    // literally mention the name of this type alias. Add a test for this.
+    // (also maybe double check this logic?)
+    // TODO(jez) There's actually some non-trivial logic in resolver to handle constant fields.
+    // They're no longer handled by resolveSigs; they're handled by ResolveTypeMembersAndFieldsWalk
+    // now. Stare at that code to see if this whole idea is not going to work.
```

Comment:
```suggestion
```

I think we can just delete this comment and move the discussion to resolver.cc where you already commented.